src/views/Team/components/Footer/index.view.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { ReactComponent as Arrow } from "images/TeamPage/arrow.svg";
+import "./Footer.scss";
+
+const Footer: React.FC = () => {
+  return (
+    <div className="HeroFooter">
```

Comment:
Would there be SCSS naming conflicts if this class was changed to just `Footer`? I'm thinking there would be but there must be a way to contextualize  this for different pages.

Reply:
Unfortunately, I think there is. Not entirely sure why, but when I had the `className` = `Footer`, the styling of the footer in the `Home` view seemed to port over. 

<img width="1425" alt="Screen Shot 2021-01-11 at 9 36 23 PM" src="https://user-images.githubusercontent.com/40477553/104274146-0f43ff80-5455-11eb-95b5-5382d62bd5e0.png">

Which is not what I specified and what's on the Figma.

Reply:
React probably ports all files as a single SCSS resource, which causes naming conflicts. Usually this is resolved by component/style nesting, but we aren't able to do that because of our nesting depth specifications. 

Sounds good though!